docker/Dockerfile

```diff
@@ -2,22 +2,7 @@
 # This Dockerfile builds cstor istgt container running istgt from istgt base image
 #
 
-FROM ubuntu:14.04
-RUN apt-get update; exit 0
-RUN apt-get -y install rsyslog
-RUN apt-get -y install net-tools iputils-ping
-RUN apt-get -y install libssl-dev libjson-c-dev libjemalloc-dev
-RUN apt -y install apt-file && apt-file update
-
-RUN mkdir -p /usr/local/etc/bkpistgt
-RUN mkdir -p /usr/local/etc/istgt
-COPY istgt istgtcontrol /usr/local/bin/
-COPY istgt.conf istgtcontrol.conf /usr/local/etc/bkpistgt/
-RUN touch /usr/local/etc/bkpistgt/auth.conf
-RUN touch /usr/local/etc/bkpistgt/logfile
-
-COPY entrypoint-istgtimage.sh /usr/local/bin/
-RUN chmod +x /usr/local/bin/entrypoint-istgtimage.sh
+FROM ubuntu:16.04
```

Comment:
@Moteesh did you noticed any issues by having ubuntu 16 for istgt docker image? (or) are we good to use ubuntu 16?

Reply:
I got info from @Moteesh that we are good to use ubuntu 16, as long as, we build the image from ubuntu 16